android/quest/src/main/java/org/smartregister/fhircore/quest/data/patient/PatientRepository.kt

```diff
@@ -149,7 +149,8 @@ constructor(
 
       result.map {
         QuestionnaireConfig(
-          appId = configurationRegistry.appId,
+          appId =
+            if (configurationRegistry.isAppIdInitialized()) configurationRegistry.appId else "",
```

Comment:
can we set to null instead of empty string if uninitialized? and update QuestionnaireConfig to handle that
```suggestion
            configurationRegistry.isAppIdInitialized() and configurationRegistry.appId,
```

Reply:
Remove this from here and would handle this in #955

Reply:
are you gong to revert the change here?